tests/ui/impl-trait/in-trait/auxiliary/rpitit.rs

```diff
@@ -5,10 +5,10 @@
 use std::ops::Deref;
 
 pub trait Foo {
-    fn bar() -> impl Deref<Target = impl Sized>;
+    fn bar(self) -> impl Deref<Target = impl Sized>;
```

Comment:
Adding a self-param for this aux crate shared between the two tests I touched below, so I can use it for a `dyn` error :smile_cat: